lxd/response/response.go

```diff
@@ -445,6 +445,11 @@ func (r *fileResponse) Render(w http.ResponseWriter) error {
 
 	for _, entry := range r.files {
 		var rd io.Reader
+
+		if entry.Cleanup != nil {
```

Comment:
@MusicDin I think we should leave this one alone, as in cases of multiple files I dont think its a good idea to stack up individual file cleanup functions as that changes the order of when things are run.